cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -1986,6 +1990,185 @@ def test_unmanaged_fsx_multiple_az_validator(
+                    name="different-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1"],
+                    ),
+                ),
+            ],
+            [{"subnet-1": "us-east-1a"}, {"subnet-1": "us-east-1a"}],
+            FailureLevel.INFO,
+            [
+                "Your configuration for Queue 'different-az-queue-1' includes multiple subnets and external "
+                + "shared storage configuration. Accessing a shared storage from different AZs can lead to increased "
+                + "latency and costs.",
+                "Your configuration for Queue 'different-az-queue-2' includes multiple subnets and external "
+                + "shared storage configuration. Accessing a shared storage from different AZs can lead to increased "
+                + "latency and costs.",
+            ],
+        ),
+        (
+            "us-east-1a",
+            "us-east-1b",
+            [
+                SlurmQueue(
+                    name="same-az-queue-1",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1"],
+                    ),
+                ),
+                SlurmQueue(
+                    name="same-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1"],
+                    ),
+                ),
+            ],
+            [{"subnet-1": "us-east-1a"}, {"subnet-1": "us-east-1a"}],
+            FailureLevel.ERROR,
+            [
+                "Your configuration includes an EBS volume created in a different availability zone than the "
+                + "Head Node. The volume and instance must be in the same Availability Zone.",
+            ],
+        ),
+        (
+            "us-east-1a",
+            "us-east-1b",
+            [
+                SlurmQueue(
+                    name="same-az-queue-1",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-1"],
+                    ),
+                ),
+                SlurmQueue(
+                    name="different-az-queue-2",
+                    compute_resources=[],
+                    networking=SlurmQueueNetworking(
+                        subnet_ids=["subnet-2"],
+                    ),
+                ),
+            ],
+            [{"subnet-1": "us-east-1a"}, {"subnet-2": "us-east-1b"}],
+            FailureLevel.ERROR,
+            [
+                "Your configuration includes an EBS volume created in a different availability zone than the "
+                + "Head Node. The volume and instance must be in the same Availability Zone.",
+                "Your configuration for Queue 'different-az-queue-2' includes multiple subnets and external "
```

Comment:
not sure if the info message is useful in this case, since it comes from an invalid configuration. What about returning only the error message?